.github/workflows/integration.yml

```diff
@@ -213,29 +210,37 @@ jobs:
         tar -zxvf kubebuilder_${version}_linux_${arch}.tar.gz
         mv kubebuilder_${version}_linux_${arch} kubebuilder && sudo mv kubebuilder /usr/local/
         export PATH=$PATH:/usr/local/kubebuilder/bin
```

Comment:
This line could be dropped as (if I am not wrong) it is not persisted to next step